front/src/modules/simulationResult/helpers/utils.ts

```diff
@@ -0,0 +1,34 @@
+import type { LayerRangeData } from '../types';
+
+const interpolateRange = (
```

Comment:
Right, I'm pleading guilty here. I picked this name because originally this function only did some interpolation and nothing else, but now it does quite a bit more. New name sounds good to me (obligatory alternative name for the bikeshed: `cutSpaceTimeRect`).

Reply:
Ok let's go for @emersion proposal. And moved the util in space time chart folder